File: Php/core/Analytics/GoogleAnalyticsModel.php
namespace core\Analytics{

	use Mishiin\Environment;

	class GoogleAnalyticsModel{

		public $enableAnalytics;

		public function __construct(){
			$this->enableAnalytics = $this->isInProduction();
		}

		private function isInProduction(){
			return Environment::getHost() == 'pokerbunch.com';
		}

	}

}

File: Web/Models/GoogleAnalyticsModel.cs
namespace Web.Models{

	public class GoogleAnalyticsModel{

	    public bool EnableAnalytics { get; private set; }

	    public GoogleAnalyticsModel()
	    {
	        EnableAnalytics = IsInProduction();
	    }
        
		private bool IsInProduction()
		{
		    return false; //Environment::getHost() == 'pokerbunch.com';
		}

	}

}